staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/envelope.go

```diff
@@ -94,13 +94,14 @@ func (t *envelopeTransformer) TransformFromStorage(data []byte, context value.Co
 		value.RecordCacheMiss()
 		key, err := t.envelopeService.Decrypt(encKey)
 		if err != nil {
-			return nil, false, fmt.Errorf("error while decrypting key: %q", err)
+			return nil, false, err
```

Comment:
We should add a comment that explains why we don't wrap (add more information) to the error message.

Reply:
`status.FromError` is producing status.Status.
`status.FromError(err).Err() == err`

Reply:
The %q should definitely be a %v. As of go1.13 which we are upgrading to [imminently](https://github.com/kubernetes/kubernetes/issues/82531), you can call https://godoc.org/errors#As. Are we using status.FromErr yet?

Reply:
yep, https://godoc.org/errors#As can be used to unwrap the error before we record the metric. 

no, we are not using status.FromError.